Makefile

```diff
@@ -290,3 +289,15 @@ pg-demo-coord-build-app: export TAGS=latest
 pg-demo-coord-build-app:
 	$(MAKE) build-target
 	$(MAKE) push-target
+
+.PHONY: release
+## Cut a new release. Example: make release TAG=v1.0.6
+release:
```

Comment:
it would make sense to add a 

```if [ -f ./notes/NOTES.$(TAG).md ]; then 1>&2 echo 'release notes not found' && exit 1; fi```

or at least

```if [ -f ./notes/NOTES.$(TAG).md ]; then 1>&2 echo "${YELLOW}release notes not found${RESET}" && echo 'enter to continue' && read; fi```

to this target to warn early?

Reply:
generating release notes / changelog remains a todo at this moment. I wouldn't make it a requirement. I've just prepared the stage in CI to push custom release notes.

Reply:
whether it's a requirement or not, i'm okay with the decision.

my take is that it's a nice feature that'd be easy to forget about, so i'd prefer to get a difficult-to-miss warning in case release notes aren't found.